src/app/state/snapshot/snapshot.reducer.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { initialState, reducer } from './snapshot.reducer';
+
+xdescribe('Stake Reducer', () => {
```

Comment:
Is this meant to be skipped? If so, maybe add a comment as to why it is skipped

Reply:
Nah, it's just copy pasted. I should remove it, as for now i'm not planning write unit tests for reducers (for now they are only for storing data, not to manipulate).